bug-triage/team-trackers.md

```diff
@@ -0,0 +1,341 @@
+
+RC channel:
+* [#navigation](https://chat.godotengine.org/channel/navigation)
+
+Git Hub team:
+* [godotengine/navigation](https://github.com/godotengine/godot/pulls?q=is%3Apr+is%3Aopen+team-review-requested%3Agodotengine%2Fnavigation)
+
+GitHub label:
+* `topic:navigation` [issues](https://github.com/godotengine/godot/issues?q=is%3Aissue%20state%3Aopen%20label%3Atopic%3Anavigation) [PRs](https://github.com/godotengine/godot/pulls?q=is%3Apr+is%3Aopen+label%3Atopic%3Anavigation)
+
+### .NET / Mono
+
+Tracked in:
+* [.NET Issue Triage](https://github.com/orgs/godotengine/projects/83) (Internal tracker)
+
+RC channel:
+* [#dotnet](https://chat.godotengine.org/channel/dotnet)
+
+Git Hub team:
+* [godotengine/dotnet](https://github.com/godotengine/godot/pulls?q=is%3Apr+is%3Aopen+team-review-requested%3Agodotengine%2Fdotnet)
+
+GitHub label:
+* `topic:dotnet` [issues](https://github.com/godotengine/godot/issues?q=is%3Aissue%20state%3Aopen%20label%3Atopic%3Adotnet) [PRs](https://github.com/godotengine/godot/pulls?q=is%3Apr+is%3Aopen+label%3Atopic%3Adotnet)
+
+### Network
+
+Tracked in:
+* [Network Issue Triage](https://github.com/orgs/godotengine/projects/96)
+
+RC channel:
+* [#networking](https://chat.godotengine.org/channel/networking)
+
+Git Hub team:
+* [godotengine/network](https://github.com/godotengine/godot/pulls?q=is%3Apr+is%3Aopen+team-review-requested%3Agodotengine%2Fnetwork)
+
+GitHub labels:
+* `topic:multiplayer` [issues](https://github.com/godotengine/godot/issues?q=is%3Aissue%20state%3Aopen%20label%3Atopic%3Amultiplayer) [PRs](https://github.com/godotengine/godot/pulls?q=is%3Apr+is%3Aopen+label%3Atopic%3Amultiplayer)
+* `topic:network` [issues](https://github.com/godotengine/godot/issues?q=is%3Aissue%20state%3Aopen%20label%3Atopic%3Anetwork) [PRs](https://github.com/godotengine/godot/pulls?q=is%3Apr+is%3Aopen+label%3Atopic%3Anetwork)
+
+### Particles
+
+Tracked in:
+* [Particles Issue Triage](https://github.com/orgs/godotengine/projects/115)
+
+RC channel:
+* [#vfx-tech-art](https://chat.godotengine.org/channel/vfx-tech-art)
+
+Git Hub team:
+* No dedicated team (currently)
+
+GitHub label:
+* `topic:particles` [issues](https://github.com/godotengine/godot/issues?q=is%3Aissue%20state%3Aopen%20label%3Atopic%3Aparticles) [PRs](https://github.com/godotengine/godot/pulls?q=is%3Apr+is%3Aopen+label%3Atopic%3Aparticles)
+
+### Physics
+
+Tracked in:
+* [Physics Issue Triage](https://github.com/orgs/godotengine/projects/102)
+
+RC channel:
+* [#physics](https://chat.godotengine.org/channel/physics)
+
+Git Hub team:
+* [godotengine/physics](https://github.com/godotengine/godot/pulls?q=is%3Apr+is%3Aopen+team-review-requested%3Agodotengine%2Fphysics)
+
+GitHub label:
+* `topic:physics` [issues](https://github.com/godotengine/godot/issues?q=is%3Aissue%20state%3Aopen%20label%3Atopic%3Aphysics) [PRs](https://github.com/godotengine/godot/pulls?q=is%3Apr+is%3Aopen+label%3Atopic%3Aphysics)
+
+### Platforms
```

Comment:
thought: If you do the tables you could potentially give Platforms their own table